app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt

```diff
@@ -53,7 +53,7 @@ class LoginActivity : AppCompatActivity(), ILoginView {
         loginPresenter.onAttach(this)
     }
 
-    override fun onLoginSuccess(message: String) {
+    override fun onLoginSuccess(message: String?) {
```

Comment:
Yes because without "?" , I have to append "!!" at the end of the string while calling the function